src/components/EntityOpsLevelMaturityContent.tsx

```diff
@@ -59,10 +60,92 @@ export const EntityOpsLevelMaturityContent = () => {
   }
 
   const { maturityReport } = service;
-  const levels = opsLevelData.account?.rubric?.levels?.nodes;
-  const levelCategories = opsLevelData.account?.service?.maturityReport?.categoryBreakdown;
-  const checkResultsByLevel = opsLevelData.account?.service?.serviceStats?.rubric?.checkResults?.byLevel?.nodes;
-  const checkStats = opsLevelData.account?.service?.checkStats;
+  const levels = opsLevelData.account.rubric?.levels?.nodes;
+  const levelCategories = opsLevelData.account.service.maturityReport?.categoryBreakdown;
+  const checkResultsByLevel = opsLevelData.account.service.serviceStats?.rubric?.checkResults?.byLevel?.nodes;
+  const scorecards = opsLevelData.account.service.serviceStats?.scorecards?.nodes;
+  // const checkStats = opsLevelData.account.service.checkStats;
+  const scorecardCategories = scorecards ? scorecards.reduce(
+    (scoreCardAccumulator: LevelCategory[], currentScorecard) => {
+      if (!currentScorecard.categories?.edges) return scoreCardAccumulator;
+      return [
+        ...scoreCardAccumulator,
+        ...currentScorecard.categories.edges.reduce(
+          (nodeAccumulator: LevelCategory[], currentEdge): LevelCategory[] => [
+            ...nodeAccumulator,
+            {
+              category: {name: currentEdge.node?.name ?? ''},
```

Comment:
First time I've seen `??` in a codebase I'm working on, neat!

Reply:
I am so used to `||` that it looks weird to me, but it's the better thing for sure.